src/components/Reports/ViewPhotos.tsx

```diff
@@ -1,35 +1,64 @@
-import { Box, useTheme } from '@mui/material';
+import { Box, Theme, useTheme } from '@mui/material';
 import { useEffect, useState } from 'react';
 import useSnackbar from 'src/utils/useSnackbar';
 import ViewPhotosModal from 'src/components/common/ViewPhotosModal';
 import ReportService, { REPORT_PHOTO_ENDPOINT } from 'src/services/ReportService';
+import { Button } from '@terraware/web-components';
+import { makeStyles } from '@mui/styles';
+import { ReportPhoto } from 'src/types/Report';
 
 type PhotosSectionProps = {
   reportId: number;
+  onPhotoRemove?: (id: number) => void;
+  editable: boolean;
 };
 
-export default function ViewPhotos({ reportId }: PhotosSectionProps): JSX.Element {
+type ReportPhotoWithUrl = ReportPhoto & { url: string };
+
+const useStyles = makeStyles((theme: Theme) => ({
+  removePhoto: {
```

Comment:
don't want to create more work for you but seems like we can extract out a component to reuse between report and withdraw batch photos (same display to remove a photo)?
if it's a big risk .. we can follow up with a cleanup ticket

Reply:
From the withdrawal log we can't remove photos of the withdrawals. We can remove photos when creating a new withdraw and uploading photos, and we are reusing that component to upload here too. Is that what you mean?

Reply:
the display only.. attaching screenshot 
<img width="360" alt="Terraware App 2023-03-08 12-13-36" src="https://user-images.githubusercontent.com/1865174/223839024-63cf1193-f4d1-4d62-8c48-410f6c6228a2.png">

Reply:
this deserves to be a separate ticket.. lets just create one and not do it here..